testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -97,12 +97,12 @@ def test_load_trajectory_stringio(self):
         u = mda.Universe(StringIO(CHOL_GRO), StringIO(CHOL_GRO),  format='GRO', topology_format='GRO')
         assert_equal(len(u.atoms), 8, "Loading universe from StringIO failed somehow")
 
-    def test_make_universe_no_args(self):
```

Comment:
So this is now broken and replaced with `mda.Universe(None)`? If so CHANGELOG it